File: sindbad_new/model/modules/getStates/getStates_simple/getStates_simple.m
function [fx,s,d,f] = getStates_simple(f,fe,fx,s,d,p,info,tix)
% #########################################################################
% PURPOSE	: initialize the states for the current time steps from the
% d.tmp.... variable
% 
% REFERENCES:
% 
% CONTACT	: ncarval; mjung
% 
% INPUT
% 
% OUTPUT
% 
% NOTES:
% 
% #########################################################################

% water pools
% s.w.wSoil(:,tix)  = s.prev.wSM;
% s.w.wGW(:,tix)  = s.prev.wGW;
% s.wd.wGWR(:,tix) = s.prev.wGWR;
% s.w.wSnow(:,tix) = s.prev.wSWE;
% s.wd.WTD(:,tix) = s.prev.WTD;

% Water Balance Pool
s.wd.WBP  = f.Rain(:,tix);



end % function

File: src/Models/getStates/getStates_simple.jl
@with_kw struct getStates_simple{T} <: getStates
    updateState::T = true
    # wSoil::type = 0.0
    # wSnow::type = 0.0
end

function compute(o::getStates_simple, forcing, out)
    @unpack_getStates_simple o
    (; rain) = out
    WBP = rain
    return (; out..., WBP)
end

function update(o::getStates_simple, forcing, out)
    return out
end

export getStates_simple